app/graphql/types/pending_message_type.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Types
+  class PendingMessageType < Types::BaseObject
+    field :id, ID, null: false
+    field :election, Types::ElectionType, null: true
```

Comment:
Could the election be `null` for some kind of message?

Reply:
yes, at the moment we don't know the election of the message until we process it. We will change this when we add the `unique_id` for messages, where we will add the election id and we will be able to get it without having to decode the message.